swayidle.1.scd

```diff
@@ -98,6 +98,16 @@ It will also lock your screen before your computer goes to sleep.
 To make sure swayidle waits for swaylock to lock the screen before it releases the
 inhibition lock, the *-w* options is used in swayidle, and *-f* in swaylock.
 
+# EXAMPLE CONFIG
```

Comment:
Rather than a new section, maybe just write something like:

```
The events can alternatively be put into a configuration file as follows:

...

If a default location is chosen, the swayidle instantiation is then simply `swayidle -w`.
```

Reply:
Wait, where's the `-w` coming from? Is that useful or recommended if you're not using `before-suspend`?